Tests/MapboxNavigationTests/SKUTests.swift

```diff
@@ -6,6 +6,7 @@ import MapboxSpeech
 import MapboxNavigation
 import MapboxCommon_Private
 
+#if DEBUG
```

Comment:
Could you also add a comment on why we don't run some tests in the `Release` configuration?

Reply:
Because the billing API used in these tests was already wrapped in #if DEBUG, for security reasons.